src/utils.ts

```diff
@@ -292,3 +293,6 @@ export const findLastIndex = <T>(
   }
   return -1;
 };
+
+export const isBackgroundTransparent = (backgroundColor: string) =>
+  backgroundColor === BACKGROUND_COLORS.TRANSPARENT;
```

Comment:
sombody would argue that `#RRGGBBAA`, where `AA` == `00` is also transparent.

Reply:
Let's factor this one out and use it instead:

https://github.com/excalidraw/excalidraw/blob/2de7f73a7119c60754bf28039a0ca6dce9aa2142/src/renderer/renderScene.ts#L213-L217